bundles/org.eclipse.passage.lic.equinox/src/org/eclipse/passage/lic/equinox/access/ConsoleInteraction.java

```diff
@@ -41,4 +49,14 @@ public void swear(Throwable thro) {
 		thro.printStackTrace(System.err);
 	}
 
+	private int stop() {
+		if (Platform.getOS().equals(Platform.OS_LINUX)) {
+			return 10;
+		} else if (Platform.getOS().equals(Platform.OS_MACOSX)) {
+			return 10;
+		} else {
+			return 13;
```

Comment:
I would process OS_WINDOWS case explicitly and then I'd use `10` for unknown

Reply:
Done